src/client/src/tracer_client.rs

```diff
@@ -279,21 +255,6 @@ impl TracerClient {
         self.process_watcher.poll_process_metrics().await
     }
 
-    #[tracing::instrument(skip(self))]
```

Comment:
same for this, what files it polls? seems to be something related to the early version of tracer @davincios @Iamdavidonuh